frontend/src/pages/_app.tsx

```diff
@@ -80,6 +80,8 @@ const ResilienceApp = ({ Component, ...rest }: AppPropsWithLayout) => {
     };
   }, [router.events]);
 
+  ga.useInitGAScript();
+
```

Comment:
```suggestion
useEffect(() => ga.useInitGAScript(), []);
```
Using useEffect we ensure it will be initialized only in client-side and once

Reply:
There is already a useEffect in the hook. Also, we need it to check if the acceptance has changed